src/main/java/com/laserfiche/api/client/model/AccessKey.java

```diff
@@ -27,6 +27,10 @@ public static AccessKey CreateFromBase64EncodedAccessKey(String base64EncodedAcc
 
             mapper.registerModule(module);
         }
+        base64EncodedAccessKey = base64EncodedAccessKey.trim();
+        if (base64EncodedAccessKey.length() == 0) {
+            throw new RuntimeException("Input cannot be empty or null");
```

Comment:
I think we could have some more sepcific exception types, like illegal argument exception (or something similar).

Reply:
ok changed it to illegal argument exception